From 9d53e3fa4516a948a6e84987e9c1c9fd07f973bf Mon Sep 17 00:00:00 2001
From: Joey Hess <joey@kitenet.net>
Date: Sun, 21 Apr 2013 15:44:51 -0400
Subject: [PATCH] static link with libxml2

This requires libxml2.a (and no .so) be installed in the ugly hardcoded
lib dir. When built this way, the haskell library will link the
C library into executables with no further options.
---
 libxml-sax.cabal |    1 +
 1 file changed, 1 insertion(+)

diff --git a/libxml-sax.cabal b/libxml-sax.cabal
index 5edfdb6..338bc55 100644
--- a/libxml-sax.cabal
+++ b/libxml-sax.cabal
@@ -31,6 +31,7 @@ library
   hs-source-dirs: lib
   ghc-options: -Wall -O2
   cc-options: -Wall
+  LD-Options: -L /home/joey/.ghc/android-14/arm-linux-androideabi-4.7/arm-linux-androideabi/sysroot/usr/lib/
 
   build-depends:
       base >= 4.1 && < 5.0
-- 
1.7.10.4

